### ISSUE TITLE
noelle c6 and kaeya c2 burst duration extend

### ISSUE THREAD

**Issue by HanSeoulOh** [genshinsim/gcsim#110]
tbd

**Comment by srliao** [comment]
waiting on #109

**Comment by HanSeoulOh** [comment]
@Tsym-or-Tysm is this issue finished?

**Comment by srliao** [comment]
no longer relevant

**Comment by srliao** [comment]
c6 noelle is actually hacked in because enemies don't die. right now user just specify how long the extension
